Framework/Source/Graphics/RenderGraph/RenderGraph.cpp

```diff
@@ -602,14 +627,12 @@ namespace Falcor
             bool isUnsatisfied = std::find(satisfiedFields.begin(), satisfiedFields.end(), field.getName()) == satisfiedFields.end();
             if (is_set(field.getType(), RenderPassReflection::Field::Type::Input) && isUnsatisfied)
             {
-                unsatisfiedInputs.push_back(passReflection.getField(i));
+                outList.push_back(passReflection.getField(i));
             }
         }
-
-        return unsatisfiedInputs;
     }
 
-    void RenderGraph::autoGenerateEdges()
+    void RenderGraph::autoGenerateEdges(const std::vector<uint32_t>& executionOrder)
```

Comment:
This is not execution order, just a random pass order. It will confuse people, since once we compile the graph we might decided to change the execution order